include/properties.hh

```diff
@@ -21,7 +21,7 @@
 #endif
 
 #ifdef TESTING
-#define NUMPARNAMES 276   // testMode command line option added
+#define NUMPARNAMES 275+2   // testMode command line option added, workingDir command line option
```

Comment:
Fine But also in compGenePred several snippets of code can be improved. For example we can subclass from CompGenePred and create three specific classes for TESTING, ESTIMATION and PREDICTION, instead of using conditional compilation. My concern here was not breaking original Augustus behavior and make testing work so that Daniel can merge his branch as well. By the way I see 275+2 is something unexpected :) I just temporarily kept it as a reminder for newly added cmd line options.

Reply:
You haven't committed the change from 275+2 to 277 in the define directive, have you?
You can do this by clicking "commit suggestion".